chain-tx-enclave-next/mls/src/ciphersuite.rs

```diff
@@ -25,7 +25,7 @@ pub enum CipherSuite {
 
 /// spec: draft-ietf-mls-protocol.md#key-schedule
 #[derive(Debug)]
-struct HKDFLabel {
+struct KDFLabel {
     // 0..255 -- hash of group context
     pub group_context: Vec<u8>,
```

Comment:
seems not in c3db39266505e9d202158d642d65918d35870e77 ?

Reply:
yes, removed